docs/EITHER.md

```diff
@@ -30,6 +30,13 @@ const success = val.right()
 const failure = 'some error'.left()
 ```
 
+### Creating an Either from an exception
+
+```javascript
+const i = 1;
+Either.fromTry(i.toPrecision(500))
```

Comment:
I was going through the source code looking for the documentation of fromTry when I found this PR. I think this line needs to be `Either.fromTry(() => i.toPrecision(500))`